airbyte-integrations/connectors/source-shopify/source_shopify/streams/streams.py

```diff
@@ -167,22 +168,16 @@ class MetafieldProducts(IncrementalShopifyGraphQlBulkStream):
     bulk_query: MetafieldProduct = MetafieldProduct
 
 
-class ProductImages(IncrementalShopifyNestedStream):
-    parent_stream_class = Products
-    nested_entity = "images"
-    # add `product_id` to each nested subrecord
-    mutation_map = {"product_id": "id"}
+class ProductImages(IncrementalShopifyGraphQlBulkStream):
```

Comment:
This streams has a lot less data in it

Also, some fields changes:
* 'created_at' goes from '2023-04-14T03:34:46-07:00' to '2023-04-14T10:34:46+00:00' which means we lose the timezone information
* position is missing
* variant_ids goes from [] to not being in the record
etc...

Should we document those and be explicit about it in the migration guide?

Reply:
> 'created_at' goes from '2023-04-14T03:34:46-07:00' to '2023-04-14T10:34:46+00:00' which means we lose the timezone information

GraphQL omits the `timezone` completely (UTC Only) we cannot do anything about it. The API returns the `2023-04-14T10:34:46Z` date-time string, which we parse to be `rfc3339` in the end, to preserve the `format` for the STATE and not having the breaking changes everywhere.

> position is missing

yes, it's not available for GraphQL.

> variant_ids goes from [] to not being in the record

yes, it's not available for GraphQL.

> Should we document those and be explicit about it in the migration guide?

I'll add the notice about it to the migration guide.

Reply:
Added to the migration-guide.